extensions/amp-experiment/amp-experiment.md

```diff
@@ -16,4 +16,90 @@ limitations under the License.
 
 # <a name="amp-experiment"></a> `amp-experiment`
 
-Under development. See [issue #1411](https://github.com/ampproject/amphtml/issues/1411).
+| | |
+|-----------------------|------------------------------|
+| **Description** | Conduct user experience experiments on an AMP document and collect corresponding data with `amp-pixel` or `amp-analytics` | 
+| **Availability** | In Experiment |
```

Comment:
"Experimental" - also link that word to https://cdn.ampproject.org/experiments.html